tests/testdata/no_empty_lines_docs.md

```diff
@@ -0,0 +1,9 @@
+{% docs aaa %}
```

Comment:
Können wir hier noch testen ob das auch funktioniert, wenn alles in einer Zeile steht und die Doku nur durch Leerzeichen von {% docs aaa %} und {% enddocs %} getrennt ist?

Reply:
Erledigt